system/jlib/jstats.cpp

```diff
@@ -983,6 +983,7 @@ static const constexpr StatisticMeta statsMetaData[StMax] = {
     { NUMSTAT(MatchRightRowsMax), "The largest number of right rows in a join group" },
     { NUMSTAT(MatchCandidates), "The number of candidate combinations of left and right rows forming join groups" },
     { NUMSTAT(MatchCandidatesMax), "The largest number of candidate combinations of left and right rows in a single group" },
+    { NUMSTAT(ParallelExecute), "The number of parallel execution of the activity" },
```

Comment:
minor: This description doesn't quite make sense.  "...parallel execution paths/threads/workers for the...".  Worth changing because this is a string presented to the user.